functions/auth.js

```diff
@@ -19,3 +30,164 @@ exports.passwordsMatch = (submittedPassword, actualPassword) =>
   submittedPassword.trim() === actualPassword.trim() ||
   lowercaseFirstChar(submittedPassword.trim()) ===
     lowercaseFirstChar(actualPassword.trim());
+
+/**
+ * Redirect the user to the authentication consent screen. The 'state' cookie is set for later state verification.
```

Comment:
```suggestion
 * Redirect the user to the authentication consent screen.
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1523/commits/28638e8502fe6cfd5e7b11b43493cb15271c4326